src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -55,7 +198,7 @@ export function subscribeComponentAnalyticsEvents(
       // If the event included additional details / context...
       if (e.detail.details) {
         for (const key of Object.keys(e.detail.details)) {
-          const newKey = `${kebabCase(e.detail.componentName)}-${key}`;
+          const newKey = `${action.prefix}-${key}`;
```

Comment:
@pdavies88 - should `action.prefix` be replaced with a destructured syntax like `version`?

```js
const { prefix } = action;
```

Reply:
@bsmartin-ep your call but I would lean towards the destructured syntax it's a bit cleaner and most linters prefer it